files/image_config/hostcfgd/hostcfgd

```diff
@@ -40,6 +40,40 @@ def obfuscate(data):
     else:
         return data
 
+
+def update_feature_status(feature_name, status):
+    if status == "enabled":
+        start_cmds = []
+        start_cmds.append("sudo systemctl unmask {}.service".format(feature_name))
+        start_cmds.append("sudo systemctl enable {}.service".format(feature_name))
+        start_cmds.append("sudo systemctl start {}.service".format(feature_name))
+        for cmd in start_cmds:
+            syslog.syslog(syslog.LOG_INFO, "Running cmd: '{}'".format(cmd))
+            try:
+                subprocess.check_call(cmd, shell=True)
+            except subprocess.CalledProcessError as err:
+                syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
```

Comment:
Is `\n` supposed to be after `{}` ?

Reply:
Good catch. I didn't modify this code, I just moved it and indented it. syslog() doesn't create a newline when it encounters `\n`. It will literally print `\n`, so this should be removed. I will add another commit to do this.

Reply:
Removed `\n` chars and shortened messages in latest commit. Thanks!